packages/serverpod/lib/src/util/string_extension.dart

```diff
@@ -0,0 +1,27 @@
+/// Extension methods for the [String] class.
+extension StringExtension on String? {
+  /// Returns true if the string is null or empty.
+  bool get isNullOrEmpty => this == null || this!.trim().isEmpty;
```

Comment:
Suggestion: Remove this method along with its negated version.

In most cases Null and Empty string have different meanings. Null is the absence of a value indicating that no string was created while empty is a valid string value that could have meaning. By providing a method like this we would encourage grouping of the two and signal that passing an empty string can have the same meaning as a Null value, which they shouldn't.

I would rather encourage explicit handling of the two in the different places.